scipy-1.0/paper.tex

```diff
@@ -618,12 +620,13 @@ \subsection*{Language choices}
 languages or major dependencies is generally unlikely---our choices are strongly
 driven by long-term stability. GPU acceleration, new transpiling libraries, and
 the latest JIT compilation approaches (i.e.,
-Numba\cite{Lam:2015:NLP:2833157.2833162}) are very powerful, but currently fall
+Numba\cite{Lam:2015:NLP:2833157.2833162}) are very powerful, but have traditionally fallen
 outside the remit of the main SciPy library. That said, we have recently
 increased our efforts to support compatibility with some of these options, and
-having our full test suite pass with the PyPy JIT
-compiler\cite{Bolz:2009:TMP:1565824.1565827} is now a requirement in our
-development workflow.
+while the exact approach we will adopt across the entire library remains unclear,
+we do now have a concrete implementation of a submodule that allows for the experimental
```

Comment:
Does "submodule" refer to `scipy.fft`? I think "subpackage" would then be better for matters of consistency.

Reply:
Alright, I've adopted "subpackage" instead of "submodule" now.